### ISSUE TITLE
NullPointerException: Attempt to invoke virtual method 'java.lang.String org.wordpress.android.fluxc.store.MediaStore$M...

### ISSUE THREAD

**Issue by AliSoftware** [wordpress-mobile/WordPress-Android#14852]
**Sentry Url**: https://sentry.io/share/issue/ade45c2c71e24ccebc29cc9280a42e15/
**User Count**: 24
**Count**: 35
**First Release**: org.wordpress.android@17.5-rc-1+1056
**First Seen**: 2021-06-08T07:01:41.667000Z
**Last Seen**: 2021-06-16T09:52:27Z
**24 Hours**: 26
**30 Days**: 36

```
NullPointerException: Attempt to invoke virtual method 'java.lang.String org.wordpress.android.fluxc.store.MediaStore$MediaError.getApiUserMessageIfAvailable()' on a null object reference
     at org.wordpress.android.ui.uploads.UploadUtils.getErrorMessageFromMediaError(UploadUtils.java:155)
     at org.wordpress.android.ui.uploads.UploadUtils.getErrorMessageFromMedia(UploadUtils.java:180)
     at org.wordpress.android.ui.posts.EditPostActivity.getErrorMessageFromMedia(EditPostActivity.java:3625)
     at org.wordpress.android.editor.gutenberg.GutenbergEditorFragment.showRetryMediaUploadDialog(GutenbergEditorFragment.java:752)
     at org.wordpress.android.editor.gutenberg.GutenbergEditorFragment.access$1400(GutenbergEditorFragment.java:82)
     at org.wordpress.android.editor.gutenberg.GutenbergEditorFragment$1.onRetryUploadForMediaClicked(GutenbergEditorFragment.java:271)
...
(8 additional frame(s) were not displayed)
```

**Comment by ashiagr** [comment]
Since the error was first seen in `17.5-rc-1 (1056)`, it could be related to the changes [here](https://github.com/wordpress-mobile/WordPress-Android/commit/e4993ba2e031b82829d7ba24b3c6f48b99232a03#diff-4036704c8d028eb045c17fdd6736a86b055b2701fbf79d4d42955455ba124c24R174-R178).

Cc @develric, in case you have any inputs.

Assigning a low priority (doesn’t affect core functionality, estimated 0-5% of users affected).

**Comment by ashiagr** [comment]
Closed via #14858.
